packages/@guardian/source-foundations/src/animation.ts

```diff
@@ -0,0 +1,11 @@
+// FYI
+// src/core/foundations/src/animation.ts SYMLINKS TO
+// packages/@guardian/source-foundations/src/animation.ts
```

Comment:
It might be worth making it more explicit that changing things here will also change the currently published packages and to therefore be careful.

Reply:
as discussed offine, boosted perms required in repo before merging 🤞